sound/soc/intel/boards/Kconfig

```diff
@@ -377,7 +377,7 @@ config SND_SOC_INTEL_SKL_HDA_DSP_GENERIC_MACH
 
 endif ## SND_SOC_INTEL_SKYLAKE_HDAUDIO_CODEC || SND_SOC_SOF_HDA_AUDIO_CODEC
 
-if SND_SOC_SOF_HDA_COMMON
+if SND_SOC_SOF_INTEL_COMMON
 config SND_SOC_INTEL_SOF_RT5682_MACH
 	tristate "SOF with rt5682 codec in I2S Mode"
 	depends on MFD_INTEL_LPSS && I2C && ACPI
```

Comment:
this dependency isn't correct for Baytrail/CHT.

Reply:
also missing 
select HDAC_HDMI if  SND_SOC_SOF_HDA_COMMON

Reply:
@plbossart Is there any side effect if we select HDAC_HDMI on Baytrail? It will be selected if we enable all platforms in the config.

Reply:
@plbossart 
There used to be some side effect if we would like to use nocodec mode but other machine driver selected HDAC_HDMI codec driver.
Need to check that for nocodec.

Reply:
@plbossart Regarding if SND_SOC_SOF_INTEL_COMMON, how about if SND_SOC_SOF_INTEL_PCI || SND_SOC_SOF_INTEL_ACPI?

Reply:
I was talking about the depends line, this should be

````
if SND_SOC_SOF_HDA_COMMON || SND_SOC_SOF_BAYTRAIL
config SND_SOC_INTEL_SOF_RT5682_MACH
	tristate "SOF with rt5682 codec in I2S Mode"	
        depends on I2C && ACPI
        depends on MFD_INTEL_LPSS if SND_SOC_SOF_HDA_COMMON
        depends on X86_INTEL_LPSS if SND_SOC_SOF_BAYTRAIL
````